examples/README.md

```diff
@@ -57,7 +57,7 @@ This Scala example demonstrates how to use Riak Spark connector to do range quer
 Before running, you will need to create and activate a TS table called `ts_weather_demo` if it does not already exist. You can find more information on creating and activating TS tables [here](http://docs.basho.com/riak/ts/latest/using/creating-activating/), or you can run the following:
 
 ```
-riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def":"CREATE TABLE ts_weather_demo (weather varchar not null, family varchar not null, time timestamp not null, temperature double, humidity double, pressure double, PRIMARY KEY ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'
```

Comment:
Looks like the underlying issue that made my test fail was some non-ASCII characters inside the create table part. If you copy and paste the first version into a file and then use "cat -v", you can see the non-ASCII characters between "temperature" and "double":

```$ cat -v create_ts_weather_demo.txt```

```riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def": "create table ts_weather_demo (weather varchar not null, family varchar not null, time timestamp not null, temperature�M-^@M-^Bdouble, humidity double, pressure double, primary key ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'```

However, this formatting fix is fine, so we can keep it.